Automattic/simplenote-android
# Feature/add app styles

### Fix
Merge the `feature/add-app-styles` branch into `develop`.   This pull request is just to merge the feature branch into the base branch to avoid merge conflicts in the future once the feature is ready to be released.  All associated changes have already been reviewed in preceding pull requests.

### Test
1. Open navigation drawer.
2. Tap **_Settings_** item in drawer.
3. Notice **_Membership_** setting under **_Account_** section.
4. Notice **_Free_** in **_Membership_** setting.
5. Notice **_Upgrade_** button in **_Membership_** setting.
6. Notice **_Style_** setting under **_Appearance_** section.
7. Notice **_Default_** in **_Style_** setting.
8. Tap **_Style_** setting under **_Appearance_** section.
9. Notice **_Default_** style is selected.
10. Notice other styles are locked.
11. Tap any style other than **_Default_**.
12. Notice **_Style Locked_** dialog is shown.
13. Tap **Cancel_** button in dialog.
14. Notice dialog is dismissed.
15. Tap any style other than **_Default_**.
16. Notice **_Style Locked_** dialog is shown.
17. Tap **_Upgrade_** button in dialog.
18. Notice https://app.simplenote.com is shown in browser (if installed).

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
